CnpSdkForNet/CnpSdkForNet/CnpOnline.cs

```diff
@@ -18,7 +18,10 @@ public class CnpOnline : ICnpOnline
         // Configuration object containing credentials and settings.
         private Dictionary<string, string> _config;
         // 
-        private Communications _communication;
+        private static Communications _communication;
+
+        // exposed merchantId for organizations with multiple Merchant Ids.
```

Comment:
@vivek42 Would there be a separate CnpOnline object for each merchant ID then? If so, my comment above may carry more weight.